releasenotes/notes/prof-3-11-traceback-issue-20e06214d4a3879b.yaml

```diff
@@ -0,0 +1,5 @@
+---
+issues:
+  - |
+    profiling: There is currently a known Python 3.11 compatibility issue where the stack collector does not properly 
```

Comment:
should we add what this potentially means for customers? E.g probably don't enable profiling on python 3.11 or else you might hit a bug?